packages/signal-client/src/newid.ts

```diff
@@ -1,4 +1,11 @@
-﻿import uuid from 'uuid'
+﻿import cuid from 'cuid'
 // use shorter ids in development & testing
-export const newid = (len: number = 4) =>
-  process.env.NODE_ENV === 'production' ? uuid() : uuid().slice(0, len)
+export const newid = (len: number = 4) => {
+  return process.env.NODE_ENV === 'production'
+    ? cuid()
+    : cuid()
+        .split('')
+        .reverse()
```

Comment:
Do I wanna know?

Reply:
see commit msg for 086914c

Reply:
I've added a comment 1832f20fcc80c6cccaf0a670fa1b2890ec02b6ae